docs/src/components/Tile.tsx

```diff
@@ -34,6 +34,36 @@ function TileTitle({ children }: { children: React.ReactNode }) {
   );
 }
 
+interface StyledContainerProps extends Pick<Props, "fullWidth"> {
+  hasContent: boolean;
+}
+
+const StyledWrapper = styled.a<StyledContainerProps>`
+  ${({ theme, fullWidth, hasContent }) => `
+    display: block;
+    padding: 2rem;
+    border-radius: 1rem;
+    background: ${theme.orbit.paletteWhite};
+    box-shadow: ${theme.orbit.boxShadowRaisedSubtle};
+    transition: box-shadow ${theme.orbit.durationFast};
+    display: flex;
+    width: ${fullWidth && "100%"};
```

Comment:
Not sure it matters really, but wouldn't it make sense as this?
```suggestion
    ${fullWidth && "width: 100%;"}
```

Reply:
it does not make any difference, if it's evaluated to `false`, the property will not be rendered.

Reply:
I was thinking in terms of reading the code rather than the result, but it probably makes no difference.

Reply:
@mainframev I recently checked, it _does_ produce the CSS declaration, but it's invalid CSS, so browsers ignore it. Ultimately it does produce the desired result, but in an uncomfortable way, so I don't think we should keep doing this.

![Screen Shot 2021-05-27 at 16 14 15](https://user-images.githubusercontent.com/471278/119842298-d9388100-bf06-11eb-8133-7b6f3393fc8c.png)

Reply:
I'll assume that it works the way you describe when using `css`, then.